src/components/sbb-toggle-check/sbb-toggle-check.e2e.ts

```diff
@@ -1,69 +1,74 @@
-    page = await newE2EPage();
-    await page.setContent('<sbb-toggle-check id="focus-id"></sbb-toggle-check>');
-    element = await page.find('sbb-toggle-check');
+    await fixture(html`<sbb-toggle-check id="focus-id"></sbb-toggle-check>`);
+    element = document.querySelector('sbb-toggle-check');
   });
 
   it('renders', async () => {
-    expect(element).toHaveClass('hydrated');
+    assert.instanceOf(element, SbbToggleCheck);
   });
 
   describe('events', () => {
     it('emit event on click', async () => {
-      await page.waitForChanges();
-      const changeSpy = await page.spyOnEvent('change');
+      await element.updateComplete;
+      const changeSpy = new EventSpy('change');
+
+      element.click();
+      await element.updateComplete;
 
-      await element.click();
       await waitForCondition(() => changeSpy.events.length === 1);
-      expect(changeSpy).toHaveReceivedEventTimes(1);
+      expect(changeSpy.count).to.be.equal(1);
     });
 
     it('should emit click on Space', async () => {
-      const changeSpy = await element.spyOnEvent('click');
+      const changeSpy = new EventSpy('click');
 
-      await element.press(' ');
-      await page.waitForChanges();
+      element.focus();
+      await sendKeys({ press: ' ' });
+      await element.updateComplete;
 
       await waitForCondition(() => changeSpy.events.length === 1);
-      expect(changeSpy).toHaveReceivedEventTimes(1);
+      expect(changeSpy.count).to.be.equal(1);
     });
 
     it('should receive focus', async () => {
-      const changeSpy = await element.spyOnEvent('focus');
+      const focusSpy = new EventSpy('focus', element);
 
-      await element.focus();
-      await page.waitForChanges();
+      element.focus();
 
-      await waitForCondition(() => changeSpy.events.length === 1);
-      expect(changeSpy).toHaveReceivedEventTimes(1);
+      await waitForCondition(() => focusSpy.events.length === 1);
+      expect(focusSpy.count).to.be.equal(1);
 
-      expect(await page.evaluate(() => document.activeElement.id)).toBe('focus-id');
+      expect(document.activeElement.id).to.be.equal('focus-id');
     });
   });
 
   it('should prevent scrolling on space bar press', async () => {
-    page = await newE2EPage();
-    await page.setContent(
-      `<div style="height: 100px; overflow: scroll" id="scroll-context">
-              <div style="height: 500px">
-                <sbb-toggle-check></sbb-toggle-check>
-              </div>
-            </div>`,
+    await fixture(
+      html`<div style="height: 100px; overflow: scroll" id="scroll-context">
+        <div style="height: 500px">
+          <sbb-toggle-check></sbb-toggle-check>
+        </div>
+      </div>`,
     );
-    element = await page.find('sbb-toggle-check');
-    expect(element).not.toHaveAttribute('checked');
-    expect(await page.evaluate(() => document.querySelector('#scroll-context').scrollTop)).toBe(0);
+    element = document.querySelector('sbb-toggle-check');
```

Comment:
I think that `element` is not the instance you expect it to be ( it probably is the one declared in the beforeEach )